encryption/classic-mceliece/classic-mceliece-kem/bench/avx_mceliece348864_avg.yaml

```diff
@@ -0,0 +1,6 @@
+timings:
+  unit: cycles
+  gen: 208145574
```

Comment:
maybe we should add a comment that this is the average (because median is vastly different).

edit, I've just noted the avg in the filename. Good idea, however, this value is just internal and not visible to the user. Only the `platform` string is.

(applies to all benchmarks)

Reply:
(another note: for now it does not matter, but please consider renaming those files from `_avg` to `_xeon`, or `_sandybridge` or something. The third part is supposed to be the architecture the measurements were taken on. The _avg is not very telling in this respect (which might become a problem once we add more measurements)